jdaviz/app.py

```diff
@@ -86,7 +86,16 @@ def equivalent_units(self, data, cid, units):
                     'erg / (s sr cm2)', 'erg / (Hz s sr cm2)',
                     'erg / (Angstrom s sr cm2)',
                     'ph / (Angstrom s sr cm2)', 'ph / (Hz s sr cm2)'
-                ])
+                ]
+                + [
```

Comment:
Do we really need the `+` operators to chain multiple static lists here? This introduces unnecessary list math, no?